src/libsync/syncfileitem.h

```diff
@@ -45,6 +45,7 @@ class SyncFileItem {
 
     enum Type {
       UnknownType = 0,
+      RequestsContainer = 1,
```

Comment:
That's the same as CSYNC_FTW_TYPE_SLINK  (which is of value 1)  So the same as SoftLink.

(Also UnkownType is the same as File.  So this enum is already broken.  But one should not make it more broken.)